apollo-adminservice/src/main/java/com/ctrip/framework/apollo/adminservice/aop/NamespaceAcquireLockAspect.java

```diff
@@ -61,40 +56,43 @@ public NamespaceAcquireLockAspect(
     this.bizConfig = bizConfig;
   }
 
-
-  //create item
-  @Before("@annotation(PreAcquireNamespaceLock) && args(appId, clusterName, namespaceName, item, ..)")
-  public void requireLockAdvice(String appId, String clusterName, String namespaceName,
-                                ItemDTO item) {
+  // create item
+  @Before(
+      "@annotation(PreAcquireNamespaceLock) && args(appId, clusterName, namespaceName, item, ..)")
+  public void requireLockAdvice(
+      String appId, String clusterName, String namespaceName, ItemDTO item) {
     acquireLock(appId, clusterName, namespaceName, item.getDataChangeLastModifiedBy());
   }
 
-  //update item
-  @Before("@annotation(PreAcquireNamespaceLock) && args(appId, clusterName, namespaceName, itemId, item, ..)")
-  public void requireLockAdvice(String appId, String clusterName, String namespaceName, long itemId,
-                                ItemDTO item) {
+  // update item
+  @Before(
+      "@annotation(PreAcquireNamespaceLock) && args(appId, clusterName, namespaceName, itemId,"
+          + " item, ..)")
```

Comment:
This style doesn't look good. Is it possible to tweak the format setting a little?
BTW, there is a [customized google java format file](https://github.com/apolloconfig/apollo/blob/master/apollo-buildtools/style/intellij-java-google-style.xml), is it possible to keep this github action consistent with that?

Reply:
Use args `--skip-reflowing-long-strings` to avoid it.

According to official website, there is no args to specify this file when formatting...

Fortunatly, in https://github.com/google/google-java-format#intellij-android-studio-and-other-jetbrains-ides we know that this tool's formatter design by https://raw.githubusercontent.com/google/styleguide/gh-pages/intellij-java-google-style.xml which same as [customized google java format file](https://github.com/apolloconfig/apollo/blob/master/apollo-buildtools/style/intellij-java-google-style.xml)